app/src/main/java/dev/arkbuilders/arkretouch/edition/manager/EditManager.kt

```diff
@@ -148,9 +164,12 @@ class EditManager {
     private val undoStack = Stack<String>()
     private val redoStack = Stack<String>()
 
+    // TODO: not used? Consider using [EditionMode] instead
     private val _isCropMode = mutableStateOf(false)
     val isCropMode = _isCropMode
 
+    var editionMode: EditionMode by mutableStateOf(EditionMode.IDLE)
```

Comment:
`EditingMode` sounds more correct. Could you create an issue for refactoring of operations as `EditingMode`s?